R/x_table_2.R

```diff
@@ -18,7 +18,7 @@ x_table_2_details <- function(x) {
 x_table_2_table_row <- function(x,details_fun) {
   unit <- NULL
   if(.has("unit",x)) {
-    unit <- glue::glue(" ({unit})", .envir = x)
+    unit <- glue::glue_data(x, " ({unit})", .envir = emptyenv())
```

Comment:
As far as I can tell, this call isn't covered by the test suite, but I checked it manually with the changes below.

<details>
<summary>diff</summary>

```diff
diff --git a/check-xtable.R b/check-xtable.R
new file mode 100644
index 0000000..8b1f119
--- /dev/null
+++ b/check-xtable.R
@@ -0,0 +1,2 @@
+devtools::load_all()
+render_define("inst/spec/project.yml")
diff --git a/inst/spec/DEM104101F_AE.yml b/inst/spec/DEM104101F_AE.yml
index 21ba705..3f7b534 100644
--- a/inst/spec/DEM104101F_AE.yml
+++ b/inst/spec/DEM104101F_AE.yml
@@ -2,6 +2,8 @@ SETUP__:
   description: AE analysis data set
   sponsor: Some other company
   data_stem: DEM0104101F_AE_2
+  glue:
+    foo: bar
 C:
   long: commented rows
   values: [".",C]
@@ -26,7 +28,7 @@ CMT:
   comment: per NONMEM specifications
   values: [1,2,3,4]
 TAFD:
-  unit: hours
+  unit: <<foo>>
   long: time after first dose
 WT:
   about: [Weight, kg]
diff --git a/inst/spec/project.yml b/inst/spec/project.yml
index a181527..7dc4e71 100644
--- a/inst/spec/project.yml
+++ b/inst/spec/project.yml
@@ -2,16 +2,16 @@
 YPROJ__:
   sponsor: ABC-Pharma
   projectnumber: ABC101104F
-  yml_file: /Users/kyleb/ghe/software/yspec/inst/spec/project.yml
-  proj_file: /Users/kyleb/ghe/software/yspec/inst/spec/project.yml
-  proj_path: /Users/kyleb/ghe/software/yspec/inst/spec
-  path: /Users/kyleb/ghe/software/yspec/inst/spec
+  yml_file: /data/home/kylem/src/github/metrumresearchgroup/yspec/inst/spec/project.yml
+  proj_file: /data/home/kylem/src/github/metrumresearchgroup/yspec/inst/spec/project.yml
+  proj_path: /data/home/kylem/src/github/metrumresearchgroup/yspec/inst/spec
+  path: /data/home/kylem/src/github/metrumresearchgroup/yspec/inst/spec
   class: yproj
 DEM104101F_PK:
   name: DEM104101F_PK
   description: Population PK analysis data set
-  spec_path: /Users/kyleb/ghe/software/yspec/inst/spec
-  spec_file: /Users/kyleb/ghe/software/yspec/inst/spec/DEM104101F_PK.yml
+  spec_path: /data/home/kylem/src/github/metrumresearchgroup/yspec/inst/spec
+  spec_file: /data/home/kylem/src/github/metrumresearchgroup/yspec/inst/spec/DEM104101F_PK.yml
   data_path: ../data/derived
   data_stem: DEM104101F_PK
   csv_file: ../data/derived/DEM104101F_PK.csv
@@ -19,8 +19,8 @@ DEM104101F_PK:
 DEM104101F_PKPD:
   name: DEM104101F_PKPD
   description: Population PKPD analysis data set
-  spec_path: /Users/kyleb/ghe/software/yspec/inst/spec
-  spec_file: /Users/kyleb/ghe/software/yspec/inst/spec/DEM104101F_PKPD.yml
+  spec_path: /data/home/kylem/src/github/metrumresearchgroup/yspec/inst/spec
+  spec_file: /data/home/kylem/src/github/metrumresearchgroup/yspec/inst/spec/DEM104101F_PKPD.yml
   data_path: ../data/derived
   data_stem: DEM104101F_PKPD
   csv_file: ../data/derived/DEM104101F_PKPD.csv
@@ -28,8 +28,8 @@ DEM104101F_PKPD:
 DEM104101F_AE:
   name: DEM104101F_AE
   description: AE analysis data set
-  spec_path: /Users/kyleb/ghe/software/yspec/inst/spec
-  spec_file: /Users/kyleb/ghe/software/yspec/inst/spec/DEM104101F_AE.yml
+  spec_path: /data/home/kylem/src/github/metrumresearchgroup/yspec/inst/spec
+  spec_file: /data/home/kylem/src/github/metrumresearchgroup/yspec/inst/spec/DEM104101F_AE.yml
   data_path: ../data/derived
   data_stem: DEM0104101F_AE_2
   csv_file: ../data/derived/DEM0104101F_AE_2.csv
```

</details>